tests/unittests/onnxImporterTest.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "ImporterTestUtils.h"
+#include "glow/ExecutionEngine/ExecutionEngine.h"
+#include "glow/Graph/Graph.h"
+#include "glow/Importer/ONNX.h"
+#include "gtest/gtest.h"
+
+using namespace glow;
+
+/// Test loading conv op from a ONNX model.
+/// The input is N*C*H*W (1*1*3*3), the kernels is {2, 2},
+/// strides is {1, 1}, pads is {1, 1, 1, 1}, group is 1.
+TEST(onnx, importConv) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetFilename("tests/models/onnxModels/simpleConv.onnxtxt");
+
+  SaveNode *output;
+  // Destroy the loader after the graph is loaded since the following execution
+  // will not depend on anyting from the loader.
+  {
+    Tensor data;
+    getImageData(&data, 1, 1, 3, 3);
+    ONNXModelLoader onnxLD(NetFilename, {"data"}, {&data}, *F);
+    output = onnxLD.getSingleOutput();
+  }
+
+  EXPECT_EQ(F->getNodes().size(), 4);
+  EXPECT_EQ(countNodeKind(F, Kinded::Kind::ConvolutionNodeKind), 1);
+  EXPECT_EQ(countNodeKind(F, Kinded::Kind::TransposeNodeKind), 2);
+  EXPECT_EQ(countNodeKind(F, Kinded::Kind::SaveNodeKind), 1);
```

Comment:
That's much better! Thanks.

I think we could even replace the counting of the three instruction with something like:
find the save node (std::find...)
Then from there, check that the input is a convolution.
Then the input of the convolution a transpose (or the other way around).

That way, we could also check that the types and dimensions of the various nodes are correct.